Dockerfile

```
@@ -1,24 +1,17 @@
-FROM alpine:3.10
-
-FROM cljkondo/clj-kondo AS clj-kondo
-
-FROM clojure AS lein
+FROM cljkondo/clj-kondo:2022.05.31-alpine AS clj-kondo
 FROM clojure:tools-deps-alpine
 
-COPY --from=clj-kondo /usr/local/bin/clj-kondo /usr/local/bin/clj-kondo
+COPY --from=clj-kondo /bin/clj-kondo /usr/local/bin/clj-kondo
```

It might be somewhat too picky, but if you don’t mean to use multi-stage builds and just want to copy files from another Docker image, [`COPY --from <image tag> <src> <dst>`](https://docs.docker.com/develop/develop-images/multistage-build/#use-an-external-image-as-a-stage) will be handy. Using that, you can remove the first `FROM` directive:

```suggestion
FROM clojure:tools-deps-alpine
COPY --from=cljkondo/clj-kondo:2022.05.31-alpine /bin/clj-kondo /usr/local/bin/clj-kondo
```